test/switch.test.js

```diff
@@ -2,11 +2,15 @@ import { Switch } from "../src/Switch"
 
 test("Switch returns only the first truthy child", () => {
   const expected = "truthy value"
-  const children = [0, null, expected, false, "other"]
-  expect(Switch(null, children)).toBe(expected)
+  const children = [0, null, expected, false, "other"].map(child => () => child)
+  expect(Switch(null, children)()).toBe(expected)
```

Comment:
Maybe we can make explicit the use of `state` and `actions` here ? 
```jsx
const state = {}
const actions = {}
expect(Switch(null, children)(state, actions)).toBe(expected)
```
Just thinking about  readability and understanding of the use of Lazy Component.